examples/client/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "client"
+version = "0.1.0"
+authors = ["Steven Joruk <steven@joruk.com>"]
+edition = "2018"
+
```

Comment:
You may want to add the `publish = false` field here

https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field

Reply:
Oh yeah, thanks